src/components/header/SearchBar.js

```diff
@@ -49,48 +70,145 @@ const styles = theme => ({
     [theme.breakpoints.up('md')]: {
       width: 200,
     },
+    [theme.breakpoints.down('xs')]: {
+      paddingLeft: theme.spacing.unit * 4,
+    },
+  },
+  menuStyle: {
+    zIndex: 1,
+    minWidth: '200px',
+    maxHeight: '200px',
+    overflow: 'auto',
   },
 });
 
-export class SearchBar extends Component {
+class SearchBar extends Component {
   constructor(props) {
     super(props);
     this.state = {
       searchValue: '',
+      open: false,
+      shownEnv: modDisplay,
     };
     this.handleSearch = this.handleSearch.bind(this);
     this.delayedSearch = debounce(this.props.searchEnvironments, 500);
     this.delayedReset = debounce(this.props.resetSearchedEnvironments, 500);
   }
 
+  handleExpandMore = () => {
+    var showMore = this.state.shownEnv;
```

Comment:
If you need to mutate a variable use `let` over `var`.

However this is also mutating state which we don't want to do. Try something like:
```
this.setState({ shownEnv: this.state.shownEnv + modDisplay });
```

Reply:
done